sklearn/model_selection/_search.py

```diff
@@ -965,11 +965,12 @@ def _store(key_name, array, weights=None, splits=False, rank=False):
             results["std_%s" % key_name] = array_stds
 
             if rank:
-                rank_result = rankdata(-array_means, method="min")
                 # when input is nan, scipy >= 1.10 rankdata returns nan. To
                 # keep previous behaviour nans are set to the
```

Comment:
I wonder if this better communicates that `np.nan` isn't set to be `min(array_means)`, since "set to the minimum value..." implies it is set to the smallest existing value. 

(May need to reformat)
```suggestion
                # keep previous behaviour nans are set to be smaller than the
```